spec/numberformat.html

```diff
@@ -49,6 +53,9 @@ <h1>InitializeNumberFormat ( _numberFormat_, _locales_, _options_ )</h1>
         1. Let _opt_ be a new Record.
         1. Let _matcher_ be ? GetOption(_options_, `"localeMatcher"`, `"string"`, &laquo; `"lookup"`, `"best fit"` &raquo;, `"best fit"`).
         1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. Let _numberingSystem_ be ? GetOption(_options_, `"numberingSystem"`, `"string"`, *undefined*, *undefined*).
+        1. If _numberingSystem_ does not match `type`, throw a *RangeError* exception.
```

Comment:
Again, "does not match type" is not implementable. I propose we follow https://tc39.github.io/proposal-intl-locale/#sec-Intl.Locale and change it to
```
        1. If _numberingSystem_ is not *undefined*, then
          1. If _numberingSystem_ does not match the `(3*8alphanum) *("-" (3*8alphanum))` sequence, throw a *RangeError* exception.

```